homeassistant/components/braviatv/media_player.py

```diff
@@ -176,16 +151,6 @@ def media_title(self):
                 return_value = f"{return_value}: {self.coordinator.program_name}"
         return return_value
```

Comment:
Same as `volume_level`: move this to the coordinator and store already calculated data.

Reply:
Done